Makefile

```diff
@@ -61,25 +67,22 @@ TAG_LATEST ?= false
 
 ifeq ($(TAG_LATEST), true)
 	IMAGE_TAGS = \
-		--tag $(IMAGE):$(VERSION) \
-		--tag $(IMAGE):latest
+		--tag "$(IMAGE):$(VERSION)" \
+		--tag "$(IMAGE)":latest
 else
 	IMAGE_TAGS = \
-		--tag $(IMAGE):$(VERSION)
+		--tag "$(IMAGE):$(VERSION)"
 endif
 
-all: manager
+all: generate fmt vet build-operator
 
 # Run tests & validate against linters
 .PHONY: check
 check: test lint-golint lint-codespell
 
 # Run tests
 test: generate fmt vet manifests
```

Comment:
Should the "all" target depend on the "manifests" target, same as the "test" target does?

Reply:
yes it should. Thanks for pointing that out